ChainRouter.php

```diff
@@ -306,4 +306,25 @@ public function getRouteCollection()
 
         return $this->routeCollection;
     }
+    
+    /**
+     * Identify if any routers have been added
+     * 
+     * @return bool
+     */
+    public function hasRouters()
+    {
+        return !empty($this->routers);
+    }
+    
+    /**
+     * Get the number of added routers
+     * 
+     * @return integer
+     */
+    public function getNumRouters()
```

Comment:
don't really see a use case for this method .. or rather it seems too specific to me imho